terraform/modules/ecs/task-definitions/sync.json.tftpl

```diff
@@ -0,0 +1,32 @@
+[{
+    "logConfiguration": {
+        "logDriver": "awslogs",
+        "secretOptions": null,
+        "options": {
+            "awslogs-group": "polytomic-sync",
+            "awslogs-region": "${region}",
+            "awslogs-stream-prefix": "service",
+            "awslogs-create-group": "true"
+        }
+    },
+    "environment": [
+        %{ for key, value in env ~}
+        {
+            "name": "${key}",
+            "value": "${value}"
+        },
+        %{ endfor ~}
+        {
+            "name": "ROLE",
+            "value": "sync"
+        }
+    ],
+    "mountPoints": [
+    {
+        "containerPath": "/var/polytomic",
+        "sourceVolume": "polytomic"
+    }],
+    "memoryReservation": 512,
```

Comment:
2048 is probably a better starting point here.

Reply:
Oh, just noticed the limits are set in the task definition. Does that mean the reservation here is a soft limit and that it'll grow to the max in the task?

Reply:
> The soft limit (in MiB) of memory to reserve for the container. When system memory is under contention, Docker attempts to keep the container memory to this soft limit. However, your container can consume more memory when needed, up to either the hard limit that's specified with the memory parameter (if applicable), or all of the available memory on the container instance, whichever comes first.

This was just me being conservative on the soft limit. It might be a good idea to bump it up slightly